tests/VolleyManagement.UnitTests/Services/GameReportService/Comparers/PivotStandingsComparer.cs

```diff
@@ -28,22 +47,24 @@ public int Compare(PivotStandingsDto x, PivotStandingsDto y)
                 Assert.Fail($"[DivisionId={x.DivisionId}] Number of team entries does not match.");
             }
 
-            if (x.GameResults.Count == y.GameResults.Count)
+            if (gameResultComparer)
             {
-                var gameResultComparer = new ShortGameResultDtoComparer();
-                for (var i = 0; i < x.GameResults.Count; i++)
+                if (x.GameResults.Count == y.GameResults.Count)
                 {
-                    if (gameResultComparer.Compare(x.GameResults[i], y.GameResults[i]) != 0)
+                    var gameResultComparer = new ShortGameResultDtoComparer();
```

Comment:
Why create object each iteration?

Reply:
I don't create object in each iteration)

Reply:
yes, no i see - it was on removed line. sorry